### DISCUSSION TITLE
Analysis of 25a and 25b

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#177] [TF5: Coupled SE/MOM6]
We started [25a](https://github.com/NCAR/amwg_dev/issues/175) and [25b](https://github.com/NCAR/amwg_dev/issues/176) based on [last week discussion](https://github.com/NCAR/amwg_dev/discussions/172). 

25a = 23 + c8 = 4.2->4.5 + start from spunup ice
25b = 23 + c8 = 4.2->4.5 + start from spunup ice/ocn

The goal (changing c8) was to make the clouds thicker. 

Julio ran the timeseries over the week-end. And Justin is going to run the ADF. 

![image (11)](https://user-images.githubusercontent.com/9723220/188658873-99e3e0c1-5bda-4f1f-95fa-f9c93bcda275.png)

The runs are now 30 years and are stopped for some analysis.

It is clear from julio's plot that RESTOM is still high and we will need another set of tuning. Let's see where SWCF and LWCF are with the tuning.

@JulioTBacmeister, @adamrher, @swrneale, @andrewgettelman, @gustavo-marques, @fobryan3, @dan800

**Comment by cecilehannay** [comment]
So far, the sea-ice in Lab sea looks ok in Julio's diags. Maybe @fobryan3 could give a more detailed analysis. Thanks.

**Comment by megandevlan** [comment]
It looks like the SWCF is more reasonable for 025a and 025b compared to 023, but still a bit high: 
![Screen Shot 2022-09-06 at 9 57 45 AM](https://user-images.githubusercontent.com/42625059/188681707-a216ecf1-52d1-451a-8007-f83c85fc106a.png)
Surface SH/LH fluxes in the Lab Sea region have gotten pretty high in 025b, but it looks like we see something similar in 022d so probably related to using a different ocean restart file. 
<img width="912" alt="image" src="https://user-images.githubusercontent.com/42625059/188682377-813f6be9-1964-4971-9ca3-0a97ea9a26c6.png">

**Comment by adamrher** [comment]
These new runs 25a/b are designed to have 23 (https://github.com/NCAR/amwg_dev/issues/158) as the control, but 23 is initialized differently. So there is no proper control for these new runs, that could be used to isolate the c8 changes in 25a/b.


That said, the 2 W/m2 reduction in SWCF is consistent with F-case /c8 experiments last week that did have a proper control for comparison. @JulioTBacmeister can you put run 23 on your RESTOM plot so we can see what c8 (+inic) did?

**Comment by justin-richling** [comment]
If it helps @adamrher here are time series from ADF w/ 023
![Coupled_ANN_TimeSeries_w_023](https://user-images.githubusercontent.com/56696811/188695701-ac2997b5-c1d7-4726-a8c1-af30c69d6b29.png)

**Reply by cecilehannay** [reply]
Thanks, Justin. I think we can drop 22c and 24 from the plot. 
Just keep 9, 16, 23, 25a and 25b.

**Comment by cecilehannay** [comment]
From Julio's [plot ](https://github.com/NCAR/amwg_dev/issues/176#issuecomment-1238412051) and [ADF diags](https://webext.cgd.ucar.edu/BWsc1850MOM/b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.025b/atm/b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.025b_23_33_vs_b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.023_23_33/):
RESTOM from 23->25b:  1.13->0.79 W/m2
SWCF from 23->25b:  41.87->43.71 W/m2
Lab sea looking ok after 33 years.

I think that there is no point to extent 25a and 25b because we are still off. Instead, I would start new simulations 26a and 26b with a second set of tuning targeting to further increase SWCF and decrease RESTOM. Maybe with gamma_coef this time.

**Reply by adamrher** [reply]
Should get more Lab Sea diags from ocean folks first before deciding next steps?

**Reply by cecilehannay** [reply]
Yes, we can wait until @fobryan3 have a chance to look at the convection before tuning to adjust SWCF.

**Reply by cecilehannay** [reply]
@adamrher 

For c8 = 4.2->4.5 
RESTOM from 23->25b: 1.13->0.79 W/m2  (-0.34 W/m2)
SWCF from 23->25b: 41.87->43.71 W/m2  (1.84 W/m2) 

If we want to change c8 further, should we go c8 = 4.8 ? Or is it outside of the range?

**Reply by cecilehannay** [reply]
I think that it will still be on the "mild" side but this is if we want to continue the strategy of babysteps.

**Reply by adamrher** [reply]
Yes, I would agree with c8=4.8 ... a linear estimate suggests this should reduce RESTOM by another 0.34 W/m2, and SWCF another 1.84 W/m2. The latter is about what we need to get a realistic SWCF, whereas the RESTOM still might be too positive (0.79 W/m2 - 0.34 W/m2 = +0.45 W/m2). Provided the lab sea diagnostics look ok in 25a/b, I'd see go for it.

**Reply by cecilehannay** [reply]
It would be good to have [Dave Bailey](https://github.com/dabail10) 's feedback on the sea-ice, with the caveat the run might be too short in terms of sea-ice variability. But at least, it would be reassuring to know there is no showstopper.

**Reply by cecilehannay** [reply]
At @adamrher's request, I added the numbers for 25a. These are similar to 25b.  

c8 = 4.2->4.5

- 25a
RESTOM from 23->25b: 1.13->0.80 W/m2 (-0.33 W/m2)
SWCF from 23->25b: 41.87->43.62 W/m2 (1.75 W/m2)

- 25b 
RESTOM from 23->25b: 1.13->0.79 W/m2 (-0.34 W/m2)
SWCF from 23->25b: 41.87->43.71 W/m2 (1.84 W/m2)


Justin added the [ADF versus obs](https://webext.cgd.ucar.edu/BWsc1850MOM/b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.025b/atm/b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.025b_23_33_vs_obs_None_None/html_img/plot_page_SWCF_ANN_LatLon_Mean.html). We might want to target the stratocumulus regions at some point. It might require gamma_coef.  

we expect that the RESTOM will be still high with c8 = 4.8. Maybe the last step would be an adjustment of gamma_coef.

**Reply by JulioTBacmeister** [reply]
I've looked at the March ocean mixed layer thickness in 25a and there are still depths 500-1000m in the Lab Sea region through to year 33.  No question it would be good to have complete ocean and sea-ice diagnostics, but I think they won't reveal a showstopper.

**Reply by cecilehannay** [reply]
Thanks @JulioTBacmeister: Should I go ahead and started 26a and 26b with c8 = 4.8? Or only one of them?

**Reply by JulioTBacmeister** [reply]
I'd say go ahead and start both 26a and 26b.

**Comment by fobryan3** [comment]
I am super busy today so have not been able to look at the solutions other
than ncview-ing the ice distribution. I will try to look more in depth at
the ocean solutions tomorrow, but agree with Juio that continuing to tune
up the radiative properties in additional runs is a good path forward.


On Tue, Sep 6, 2022 at 2:21 PM JulioTBacmeister ***@***.***>
wrote:

> I'd say go ahead and start both 26a and 26b.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NCAR/amwg_dev/discussions/177#discussioncomment-3579159>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALNOABIMEKJUSOGAD5OB5ALV46RVJANCNFSM6AAAAAAQF4GZKA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
_______________________________________________________________________
Dr. Frank O. Bryan
Oceanography Section , Climate and Global Dynamics Laboratory
National Center for Atmospheric Research, P.O. Box 3000, Boulder, CO 80307
USA
e-mail: ***@***.*** / voice : (303) 497-1394  /   fax   : (303)
497-1700

**Comment by klindsay28** [comment]
Here are a handful of plots that I've made with 25a and 25b included:

Integrated global ocean temp. This slope in these curves is directly
related to the largish RESTOM.
![image](https://user-images.githubusercontent.com/15570875/188739359-431b5a86-a70d-4dc6-b8a6-6825c6c2a155.png)

Annual and March Lab Sea average tos (SST), average sos (SSS), average
mlotst (mixed layer depth), and integrated aice (ice fraction).
I don't see show stoppers.
![image](https://user-images.githubusercontent.com/15570875/188739420-06ccbe22-cb60-41bb-9de1-eb62ecbf2356.png)
![image](https://user-images.githubusercontent.com/15570875/188739435-362f165d-71e4-4940-b2b3-f76bf947f418.png)
![image](https://user-images.githubusercontent.com/15570875/188739454-788ce172-ebda-4a7d-8f92-308c9e7bf02c.png)
![image](https://user-images.githubusercontent.com/15570875/188739717-0147730c-35ff-438b-8bca-021c46d441f6.png)
